app/views/lists/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= "#{@user.first_name} #{@user.last_name}'s wishlist for the #{@group.name} group" %></h1>
+
+<%= link_to "#{@user.first_name} #{@user.last_name}'s list", user_path(@user) %>
+<br>
+<% if @authorized_user %>
+  <%= link_to 'Create a new Wish list item', new_group_list_item_path(@group, @list) %>
```

Comment:
might look better with:
* "Add an item to your wish list"
* displaying the form so you don't have to go to a separate page

Reply:
Text changed.

>displaying the form so you don't have to go to a separate page

Moved it to an issue: #19